dask/array/core.py

```diff
@@ -3493,6 +3498,72 @@ def _vindex(x, *indexes):
     return result_1d.reshape(broadcast_shape + result_1d.shape[1:])
 
 
+def _vindex_dask_arrays(x, *indices):
+    """Point wise indexing with Dask Arrays"""
```

Comment:
Can I request a docstring or comment here?

Reply:
What else would you like to see here?

Reply:
Tried to add some prose here in the hopes of clarifying things. Please let me know what you think.